docs/06-api-reference/graph.md

```
@@ -50,9 +59,11 @@ g = Graph([process], strict_types=True)
 
 **Args:**
 - `nodes` (list[HyperNode]): List of nodes to include in the graph
-- `edges` (list[tuple] | None): Explicit edge declarations. Disables auto-inference when provided. See [Explicit Edges](#explicit-edges) below.
+- `edges` (list[tuple] | None): Explicit edge declarations. When `shared` is set, explicit edges are additive (ordering hints on top of auto-inferred edges). Otherwise, disables auto-inference when provided. See [Explicit Edges](#explicit-edges) below.
+- `entrypoint` (str | list[str] | None): Convenience shortcut for `with_entrypoint()`. Required for cyclic graphs. Accepts a node name or list of node names. See [Entrypoints](#with_entrypoint--entrypoint-narrowing) below.
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)